README.md

```diff
@@ -61,6 +61,10 @@ ros2 launch continental_ars408_socket_can.launch.xml
   - The bool parameter to determine output publishing behavior.
   - If this parameter is set to false (default value), the driver will publish output after receiving a complete cycle of sequential data from the CAN data topic.
   - If this parameter is set to true, the driver will publish output every time data is received from the CAN data topic.
+- `size_x`
+  - The size of x [m]
+- `size_y`
+  - The size of y [m]
```

Comment:
The description for size_x is not clear enough. Could you please provide more information on what this size specifically refers to ?
Could it be "The assumed size given to the output object"?

Reply:
> Could it be "The assumed size given to the output object"?

Yes.
I fixed at 15c9624